network/network.proto

```diff
@@ -0,0 +1,12 @@
+message ServerIdentity{
+    required bytes public = 1;
+    required bytes id = 2;
+    required string address = 3;
+    optional string description = 4;
+}
+
+message Roster {
+    optional bytes id = 1;
```

Comment:
1. Roster should be in Onet
2. ID is optional or required ? To me, it should be optional as it's something that can be generated but I don't really insist on it. However, it should be consistent with ServerIdentity no ? If it's required there, it should be required here ?